src/js/themes/base.js

```diff
@@ -745,6 +745,8 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       error: {
         color: 'status-critical',
         margin: { vertical: 'xsmall', horizontal: 'small' },
+        // icon: undefined
+        // iconContainer: {}, // any box props
```

Comment:
Can we have the error `container` mean the Box with `direction="row"` that contains the error message and any icon, rather than a Box just containing the icon? Spacing between the icon and message would be controlled with `gap`, and you could still use `align`.

Reply:
The reason we have put a container directly around the icon was to solve for a case where some padding/margin needed to be applied to an icon to align properly with text (depending on the text line height) -- this came up in [this design system example](https://github.com/hpe-design/design-system/blob/master/aries-site/src/examples/templates/forms/RequiredFieldsExample.js#L47). You can see the visual [at the bottom of this form](https://design-system.hpe.design/templates/forms#required-and-optional-fields). So the intent is to be able to style just an icon container and not necessary the message and icon

Reply:
Having the container around the both of them may have benefit as well -- but I wouldn't say they're solving the same thing